locations/spiders/denizbank.py

```diff
@@ -1,27 +1,26 @@
 import scrapy
-from locations.categories import Categories, apply_category
-from locations.hours import OpeningHours
-from locations.hours import DAYS
 
+from locations.categories import Categories, apply_category
 from locations.dict_parser import DictParser
 
-class DenizbankSpider(scrapy.Spider):
-    name = "denizbank"
+
+class DenizbankTrSpider(scrapy.Spider):
```

Comment:
As a nitpick, the convention I believe has been uppercase for the country codes in class names.

```class DenizbankTRSpider(scrapy.Spider):```